src/main/java/org/broadinstitute/hellbender/tools/walkers/rnaseq/ASEReadCounter.java

```diff
@@ -173,6 +174,11 @@ public void onTraversalStart() {
         }
         final String header = "contig" + separator + "position" + separator + "variantID" + separator + "refAllele" + separator + "altAllele" + separator + "refCount" + separator + "altCount" + separator + "totalCount" + separator + "lowMAPQDepth" + separator + "lowBaseQDepth" + separator + "rawDepth" + separator + "otherBases" + separator + "improperPairs";
         outputStream.println(header);
+
+        variants.stream()
+                .filter(vFeature -> ((VCFHeader)getHeaderForFeatures(vFeature)).getGenotypeSamples().isEmpty())
```

Comment:
Confirm that `getGenotypeSamples()` won't return null

Reply:
It is constructed to always be filled as a list:
```    private final List<String> mGenotypeSampleNames = new ArrayList<String>();``` 
It looks impossible for this to be null. Testing it locally it spits out a reasonable error for sites only files.